pkg/cmd/roachtest/libpq_blocklist.go

```diff
@@ -176,7 +173,17 @@ var libPQIgnorelist21_1 = libPQIgnorelist20_2
 
 var libPQIgnorelist20_2 = libPQIgnorelist20_1
 
-var libPQIgnorelist20_1 = libPQIgnorelist19_2
+var libPQIgnorelist20_1 = blocklist{
+	// TestFormatTsBacked fails due to not returning an error for accepting a
+	// timestamp format that postgres does not.
+	"TestFormatTsBackend": "41690",
+	// TestTxOptions fails because it attempts to change isolation levels.
+	"TestTxOptions": "41690",
+	// TestCopyInBinaryError is expected to error with:
+	// pq: only text format supported for COPY, however no error is returned
+	// for CRDB.
+	"TestCopyInBinaryError": "63235",
+}
 
 var libPQIgnorelist19_2 = blocklist{
```

Comment:
the test names in the ignore list here start with pq. also, we maybe can just get rid of the the 19.2 stuff and change the min version of the test to 20.1